dist2src/core.py

```diff
@@ -573,9 +614,34 @@ def convert_single_commit(self, origin_branch: str, dest_branch: str):
         # mark the last upstream commit
         self.source_git.create_tag(tag=source_git_tag, branch=dest_branch)
 
-    def add_packit_config(self, upstream_ref: str, commit: bool = False):
+    def get_lookaside_sources(self, branch: str) -> List[Dict[str, str]]:
+        """
+        Return "sources" which are meant to be set in packit.yaml so that
+        packit can fetch them from the lookaside
+        cache instead of committing them to source-git
+
+        @param branch: pick up a lookaside sources from this branch (e.g. c8 or c8s)
+        """
+        sources: List[Dict[str, str]] = []
+        if not self.lookaside_sources():
+            # the file can actually be empty
+            return sources
+        for sha, path in self.lookaside_sources():
+            url = f"https://git.centos.org/sources/{self.package_name}/{branch}/{sha}"
+            # packit is too smart, it already expects the archive to be in SPECS
+            # so we just strip the leading SOURCES dir
+            path = path.split("/")[1]
```

Comment:
I'm wondering if it would make sense to do this in `lookaside_sources()` :thinking:

Reply:
taking it back, we can't move that because copy_sources wouldn't work since it's using the paths and they need to work